packages/esm-patient-forms-app/src/forms/form-view.component.tsx

```diff
@@ -88,13 +89,13 @@ const FormView: React.FC<FormViewProps> = ({ category, forms, patientUuid, patie
       results?.map((formInfo) => {
         return {
           id: formInfo.form.uuid,
-          lastCompleted: formInfo.lastCompleted ? formatDatetime(formInfo.lastCompleted) : undefined,
+          lastCompleted: formInfo.lastCompleted ? formatDatetime(formInfo.lastCompleted) : t('never', 'Never'),
```

Comment:
Hi @icrc-jofrancisco ! Please review the changes in this file, such that it doesn't make any breaking changes to your work in #1017. 

The changes from my end only takes into consideration layout (desktop / tablet) and respective sizing.

Thanks!